CorsixTH/Lua/entities/humanoids/vip.lua

```diff
@@ -482,7 +542,45 @@ function Vip:afterLoad(old, new)
   if old < 79 then
     self.name = self.hospital.visitingVIP
   end
+  if old < 139 then
+    self.vip_rating = 8 - math.random(0,5)
+    --Make sure we only rate rooms from now on if a VIP was visiting
+    self.room_eval = 0
+    self.num_visited_rooms = 0
+    if #self.world.rooms > 79 then
+      local roll_ratio = #self.world.rooms / 40
+      local roll_ratio_rangemap = {
+      {upper = 3, value = 2},
+      {upper = 4, value = 3},
+      {upper = 5, value = 4},
+      {value = 5}
+      }
+      self.room_visit_chance = rangeMapLookup(roll_ratio, roll_ratio_rangemap)
+    else
+      self.room_visit_chance = 1
+    end
+    self.enter_patients = #self.hospital.patients - self.hospital.num_visitors + self.enter_visitors
```

Comment:
Why are you adding enter_visitors here?

Reply:
This part is trying to account for if we have more patients in the hospital than visitors. An attempt to balance self.enter_patients so the number is more realistic.